sa/sa.go

```diff
@@ -1621,6 +1621,27 @@ func (ssa *SQLStorageAuthority) getAuthorizations(ctx context.Context, table str
 		if auth.Expires == nil {
 			continue
 		}
+
+		// Retrieve challenges for the authzvar challObjs []challModel
```

Comment:
typo: "authzvar"

Reply:
For follow-up: I think we duplicate this challenge fetching code in a couple places now and should probably refactor it in later clean-up